.editorconfig

```diff
@@ -14,3 +14,6 @@ end_of_line = crlf
 
 [*.yaml]
 indent_size = 2
+
+[*.yml]
+indent_size = 2
```

Comment:
Instead you should probably remove all 2 space here and use 4 space also in yml files.

Reply:
yeah. I'll do it in the next pr/task. Currently trying to get rid of develop branch

Reply:
https://spryker.atlassian.net/browse/SDK-833